tests/sentry/api/endpoints/test_event_committers.py

```diff
@@ -28,9 +25,9 @@ def test_simple(self):
                 "fingerprint": ["group1"],
                 "timestamp": min_ago,
                 "release": release.version,
-                "stacktrace": copy.deepcopy(DEFAULT_EVENT_DATA["stacktrace"]),
             },
             project_id=project.id,
+            event_type="error",
```

Comment:
I'd recommend creating this as an enum - that'd mostly just be to help future us though. If it's an enum it would allow us to change the string from `error` to `Error` in 1 place (not a great example, but ya know...)